src/components/annotations/AnnotationsList.vue

```diff
@@ -21,40 +21,40 @@
   </div>
 </template>
 
-<script setup>
+<script setup lang="ts">
 import { computed, watch } from 'vue';
 import AnnotationIcon from '@/components/annotations/AnnotationIcon.vue';
 
-const props = defineProps({
-  activeAnnotation: {
-    type: Object,
-    default: () => {},
-  },
-  configuredAnnotations: {
-    type: Array,
-    default: () => [],
-  },
-  toggle: {
-    type: Function,
-    default: () => null,
-  },
-  types: Array,
-});
 
-const annotationTypesMapping = computed(() => (
+export interface Props {
+  activeAnnotation: ActiveAnnotation
+  configuredAnnotations: Annotation[],
+  toggle: Function,
+  types: any[]
+}
+
+const props = withDefaults(defineProps<Props>(), {
+  activeAnnotation: () => <ActiveAnnotation>{},
+  configuredAnnotations: () => <Annotation[]> [],
+  toggle: () => null,
+})
+
+const annotationTypesMapping = computed<Object>(() => (
```

Comment:
you could specify a local interface here

Reply:
Here is the link to the `AnnotationsAPI` https://subugoe.pages.gwdg.de/emo/text-api/page/annotation_specs/#target-object

Reply:
Yes, this sounds good

Reply:
added the interface `AnnotationTypesMapping`